src/vivarium_census_prl_synth_pop/utilities.py

```diff
@@ -548,3 +548,52 @@ def load_nicknames_data():
     nicknames = nicknames.apply(lambda x: x.astype(str).str.title()).set_index("name")
     nicknames = nicknames.replace("Nan", np.nan)
     return nicknames
+
+
+def extract_metadata_proportions(final_output_dir: Path) -> None:
```

Comment:
Nit: something like `record_metadata_proportions` would be a better method name.